exasol_transformers_extension/resources/templates/delete_model_udf.jinja.sql

```diff
@@ -0,0 +1,17 @@
+CREATE OR REPLACE {{ language_alias }} SET SCRIPT "TE_DELETE_MODEL_UDF"(
+    model_name VARCHAR(2000000),
+    task_type VARCHAR(2000000),
+    sub_dir VARCHAR(2000000),
+    bfs_conn VARCHAR(2000000)
```

Comment:
we use usually a different order

```suggestion
    bfs_conn VARCHAR(2000000),
    sub_dir VARCHAR(2000000),
    model_name VARCHAR(2000000),
    task_type VARCHAR(2000000)
```

Reply:
Done